RetrospectiveExtension.Frontend/package.json

```diff
@@ -9,7 +9,7 @@
     "build:p": "npm run clean && npm run compile && webpack --mode=production",
     "lint": "eslint . --ext .ts,.tsx",
     "pack:d": "npm run build:p && tfx extension create --manifests vss-extension-dev.json --rev-version --output-path ./dist/",
-    "pack:test": "npm run build:p && tfx extension create --manifests vss-extension-prod.json --output-path ./dist/testing",
+    "pack:test": "npm run compile && webpack --mode=production && tfx extension create --manifests vss-extension-prod.json --output-path ./dist/testing",
```

Comment:
is there a reason not to use build:p script?

Reply:
`Build:p` includes the `clean` command which seemingly deletes the `coverage` and `reports` folders so the later test can't successfully clear. Still need to iron out one more CI question and this should be ready to go!

Reply:
yeah, good point 👍